src/server/game/Spells/Spell.cpp

```diff
@@ -5815,6 +5815,11 @@ SpellCastResult Spell::CheckCast(bool strict)
                             (!pTempItem || !pTempItem->GetTemplate()->LockID || !pTempItem->IsLocked()))
                         return SPELL_FAILED_BAD_TARGETS;
 
+                    // We must also ensure the gameobject we are opening is still closed by the time the spell finishes.
+                    if (GameObject* go = m_targets.GetGOTarget())
+                        if (go->GetGoState() != GO_STATE_READY)
+                            return SPELL_FAILED_BAD_TARGETS;
+
```

Comment:
add brackets
```suggestion
                    if (GameObject* go = m_targets.GetGOTarget())
                    {
                        if (go->GetGoState() != GO_STATE_READY)
                        {
                            return SPELL_FAILED_BAD_TARGETS;
                        }
                    }
```

Reply:
Is that the oficial AC code aesthetic standard? I'm just asking, I'm used to TC standard and we don't use so many brackets. I can understand bracketing a dubious IF, but it's not the case here and it really leaves me wondering.

Reply:
yes it is https://www.azerothcore.org/wiki/cpp-code-standards